.circleci/config.yml

```diff
@@ -14,6 +14,7 @@ jobs:
               -G Ninja \
               -DITK_DIR:PATH=/ITK-build \
               -DBUILD_TESTING:BOOL=ON \
+              -DModule_RLEImage:BOOL=ON \
```

Comment:
Since this is the *ITKMorphologicalContourInterpolation* configuration vs the *ITK* configuration, this is going to have no effect.

I am try to see if we can build dependent packages in *ITKUltrasound*.